### PR TITLE
Multi monitor support

### PR DESCRIPTION
**Pull request by GhostTW**
Hi, This is my first PR in open source projects.
I found there is a #629 PR, but I have no idea how to add commit on it, so I create this one.
If there is anything wrong with the workflow, please tell me.

In this PR, I tried to pass all tests and implements the features.
Did I miss anything?

## Description

Implement the rest of the features that comment mentioned in #629
Fix unit tests and conflicts.

## Motivation and Context

#192

## How Has This Been Tested?

pass all unit tests and spec flow tests

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
    I guess no breaking change in this PR.

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: mRemoteV1/Properties/Settings.settings [hunk 1]
```diff
@@ -683,5 +683,8 @@
     <Setting Name="StartUpPanelName" Type="System.String" Scope="User">
       <Value Profile="(Default)">General</Value>
     </Setting>
+    <Setting Name="ConDefaultMultiMonitor" Type="System.Boolean" Scope="User">
```
Comment (sparerd) [inline mRemoteV1/Properties/Settings.settings]: You will need to also add a InDefaultMultiMonitor setting as well, to control the default inheritance.

**Review by sparerd — CHANGES_REQUESTED** [review]
I just added a page to document the changes required when adding a new property. https://github.com/mRemoteNG/mRemoteNG/wiki/Adding-a-new-connection-property

I think you have completed almost everything required - see the review comment below.

File: mRemoteV1/Properties/Settings.settings (hunk 1 above)
Comment (GhostTW) [reply mRemoteV1/Properties/Settings.settings]: Added, Thanks for review.

**Review by sparerd — APPROVED** [review]

**Comment by GhostTW** [comment]
Found two issues when UseMultiMon is turned on.
The session becomes fullscreen as default and toggles it to windowed by manual.
The session cannot display correctly. The fullscreen session displays incorrectly in one monitor, it should display with all monitors.

**Comment by qiandahao** [comment]
@GhostTW 大佬，你这跨屏最近有点头绪了么，我也遇到了相同的问题，搞不定啊。。。。。

**Comment by popofe** [comment]
> Found two issues when UseMultiMon is turned on.
> The session becomes fullscreen as default and you have to toggle it to windowed by manual.
> The session cannot display correctly. The fullscreen session displays incorrectly in one monitor, it should display with all monitors.
@GhostTW, Using the usemultimon property is not as easy as the fullscreen mode. It is up to the application to manage the container (the display area of the ActiveX) with the "ContainerHandledFullScreen" property. You also need to handle OnRequestGoFullScreen and OnRequestLeaveFullScreen events.

**Comment by sparerd** [comment]
@GhostTW Hey, just wanted to check in and see how things are going. Is there anything I can do to help keep this PR moving?

**Comment by GhostTW** [comment]
Currently mutlimon displays incorrectly.
I will try @popofe advice when I free this week.

**Comment by GhostTW** [comment]
sorry for late reply.
I couldn't find a way to make dual monitor display well on fullscreen :(

**Comment by sparerd** [comment]
@GhostTW I see what you mean. I tried playing around with the `OnRequestGoFullScreen` and `OnRequestLeaveFullScreen` as @popofe mentioned, but I'm not sure how to properly handle full screen mode with an AxHost control. I can get the total drawing area to be correct, but it still attempts to draw within the ConnectionWindow container rather than drawing across the entire monitor. The location is like (300, 150) rather than (0, 0).

**Comment by GhostTW** [comment]
I am thinking to create a simple project to test multimon.
I want to make sure that the rdp parent control not limiting the rdp control size.
Is this possible a way?

**Comment by sparerd** [comment]
Yes, it is possible but it's a little bit of work to get set up. I think the only thing you really need is a `Form` on which to display the ActiveX control

**Comment by rani-pinchuk** [comment]
Does anyone know where this issue stands? I am very interested in this feature. Thanks, Rani

**Comment by TroelsDane** [comment]
Thanks for all your great work!
I'm also very excited about this pending feature

**Comment by claudiuconstantin** [comment]
@rani-pinchuk  seems like proper multiple monitors support is more complicated to implement

**Comment by farosch** [comment]
Closing as stale

**Pull request closed without merging**